thesis/inc/technical/detailed_architecture/component_collection.tex

```diff
@@ -0,0 +1,135 @@
+\subsection{Component Collection}
+\label{subsec:detailed_component_collection}
+The component collection is where all the different building blocks
+comes together.
+As the name suggests the component collection is responsible for containing
+a collection of a specific component type.
+The collection was design for fast contiguous processing of component groups internally within the ECS,
+while giving sufficient to external lookups on individual components.
```

Comment:
while giving sufficient "missing word?" to external.
Seems like there is a word missing here.